tests/test_ops.py

```diff
@@ -1239,28 +1240,44 @@ def test_mathtransform():
+        #{"opName": "fake_quant_with_min_max_args", "outName": "fake_quant/min_max_args/rank1_1_5_4bit_narrow", "varShapes":[[5,5]], "varTypes":["float32"], "varInit":["uniform10"], "num_bits":4, "narrow_range":True, "min":1.0, "max":5.0},
 
         # {"opName": "fake_quant_with_min_max_vars_per_channel", "outName": "fake_quant/min_max_args_per_channel/rank1_8bit", "varShapes":[[5], [5], [5]], "varTypes":["float32", "float32", "float32"], "varInit":["uniform", "uniform_m1_0", "uniform"], "num_bits":8, "narrow_range":False},
         # {"opName": "fake_quant_with_min_max_vars_per_channel", "outName": "fake_quant/min_max_args_per_channel/rank2_8bit_narrow", "varShapes":[[3, 5], [5], [5]], "varTypes":["float32", "float32", "float32"], "varInit":["uniform", "uniform_m1_0", "uniform"], "num_bits":8, "narrow_range":True},
         # {"opName": "fake_quant_with_min_max_vars_per_channel", "outName": "fake_quant/min_max_args_per_channel/rank4_6bit", "varShapes":[[3, 2, 2, 5], [5], [5]], "varTypes":["float32", "float32", "float32"], "varInit":["uniform", "uniform_m1_0", "uniform"], "num_bits":6, "narrow_range":False},
 
+         #{"opName": "adjust_contrast", "outName": "adjust_contrast/rank3_float32", "varShapes": [[8,8,3], []],"varTypes": ["float32", "float32"], "varInit": ["uniform", "uniform", "two"]},
+         #{"opName": "adjust_contrast", "outName": "adjust_contrast_v/rank3_mixed", "varShapes": [[8,8,3,1], []],"varTypes": ["float32", "float32"], "varInit": ["uniform", "two"]},
+         #{"opName": "adjust_contrast", "outName": "adjust_contrast_v/rank4_mixed","varShapes": [[8, 8, 3, 4], []], "varTypes": ["float32", "float32"], "varInit": ["uniform", "two"]},
+
+        #{"opName": "crop_and_resize", "outName": "crop_and_resize", "varShapes": [[1,2,2,1], [2,4], [2], [2]],"varTypes": ["float32", "float32","int32","int32"], "varInit": ["uniform", "uniform","uniform_int10","uniform_int10"]},
+
+
+        # {"opName": "draw_bounding_boxes", "outName": "draw_bounding_boxes", "varShapes": [[2,256,256,1], [2,2,4]],"varTypes": ["float32", "float32"], "varInit": ["uniform", "uniform"]},
+
+        # {"opName": "resize_bilinear", "outName": "resize_bilinear/float32", "varShapes": [[2,256,256,1], [2]],"varTypes": ["float32", "int32"], "varInit": ["uniform", "uniform_int2"]},
+        # {"opName": "resize_bilinear", "outName": "resize_bilinear/float64", "varShapes": [[2, 256, 256, 1], [2]],  "varTypes": ["float64", "int32"], "varInit": ["uniform", "uniform_int2"]},
+
+        # {"opName": "resize_nearest_neighbor", "outName": "resize_nearest_neighbor/float32", "varShapes": [[2,256,256,1], [2]],"varTypes": ["float32", "int32"], "varInit": ["uniform", "uniform_int10"]},
+        #  {"opName": "resize_nearest_neighbor", "outName": "resize_nearest_neighbor/float64", "varShapes": [[2,256,256,1], [2]],"varTypes": ["float64", "int32"], "varInit": ["uniform", "uniform_int10"]},
```

Comment:
as per resize_bilinear (except for init which is OK)

Reply:
Fixed.